### ISSUE TITLE
Does anyone have access to the original docs?

### ISSUE THREAD

**Issue by jeffmikels** [trueinteractions/tint2#203]
The Documentation website is down... does anyone have access to what was there?

**Comment by trevorlinton** [comment]
Yeah, it can be generated from the sources, I'll see if we can get it on a GitHub.io soon

**Comment by mdynnl** [comment]
Just in case someone got here, the docs is at https://github.com/trueinteractions/tint-docs.
`git clone`, `cd` and `python -m http.server build`.
